decidim-core/spec/presenters/decidim/push_notification_presenter_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe PushNotificationPresenter, type: :presenter do
+    let(:notification) { create(:notification) }
+    let(:np) { described_class.new(notification) }
```

Comment:
Hi guys! I'm reviewing backports and just found this line that can't find out what it means. I removed it locally and rerun this spec, and it passes. Is there something that I'm missing @ferblape?

Reply:
It was a mistake, the variable is not used in the spec.

Reply:
welcome back!